src/image-viewer.js

```diff
@@ -240,6 +240,10 @@ export class ImageViewer {
       // Set src provisionally to the known loaded value for fast display.
       // It will be updated later.
       this.image_.setAttribute('src', sourceImage.src);
+    } else {
+      // Most browsers show a gray border for images with alt but no src
+      // If we did not set the src, remove the alt attribute
+      this.image_.removeAttribute('alt');
```

Comment:
How does it get set initially? Is it only through `updateSrc_`?

Reply:
It gets set initially in the `init` function here (this code is in the `init` function) IFF the image has already loaded. Then in `updateSrc` it gets set again. The basic idea is that you set the already loaded image src in `init`, and then you upgrade it to the best fit version from `srcSet` in `updateSrc`. `updateSrc` is also called after every `measure()`, since `measure` updates the height and width of the image. But if the image wasn't loaded, then you shouldn't be setting the src here. 

But this behavior definitely raises the question of a perf bug... all of these images should have already been loaded by the time you click on them to load the lightbox. Although there are alternate ways of triggering the lightbox, which makes the code here reasonable, there's almost definitely something wrong with the way we're detecting whether an image has loaded here... =.="

Reply:
Yeah, actually it's because we are not passing in the `sourceImage` parameter to `init`. This was an oversight in my original port of the implementation. Testing in progress, fix incoming.